### DISCUSSION TITLE
License File Release

### DISCUSSION THREAD

**Discussion by endink** [endink/Mediapipe4u-plugin#82] [Announcements]
## License File Release
---   

<mark>旧版本已经不再支持，`MediaPipe4U` 版本必须高于 `20250401`, 移步这里获得许可证文件:    <mark>      

<mark>Older versions are no longer supported. The `MediaPipe4U` version must be higher than `20250401`. Go here to obtain the license file:</mark>

> 每月**1 号**早晨**8 点**自动发布新的许可证文件 。
>  New license files are automatically released at **8 a.m.** on the **1st** of every month

https://github.com/endink/Mediapipe4u-plugin/releases/tag/license-auto-release

### 如何安装许可证 (How to Install License file) ?    
English  Refer:     
https://opensource.labijie.com/Mediapipe4u-plugin/en/licensing/update_license/ 

中文参考：    
https://opensource.labijie.com/Mediapipe4u-plugin/licensing/update_license

**Comment by wanlilu** [comment]
please public source code!

**Comment by wanlilu** [comment]
Buy inertial motion capture suits if people have money.

**Comment by AngyIce** [comment]
new license file ? thank you

**Comment by sautechgroup** [comment]
Hello @endink I sent you an email to sharping@outlook.com for license informations.

**Comment by endink** [comment]
@AngyIce @sautechgroup Uploaded !

**Reply by AngyIce** [reply]
thanks man !!

**Reply by sautechgroup** [reply]
Hello @endink we are interested to buy a commercial license. I sent you an email to sharping@outlook.com about this yesterday. Thank you

**Reply by endink** [reply]
@sautechgroup The email address is correct. But I didn't receive your email, please send email again.

**Reply by sautechgroup** [reply]
I sent you another mail

**Comment by Roshan200302** [comment]
When I add body animation asset to Metahuman. At the time of lipsync, Metahuman head is getting detached. Please help me to fix this issue

**Comment by qinsong729** [comment]
2023/12/28 license has expired.

**Comment by HiWenHao** [comment]
等一个新日期的许可证😊

**Comment by endink** [comment]
Happy Chinese year, license updated , it will expired after Chinsese New Year (2024-02-28) 🛩️

**Reply by qinsong729** [reply]
thank you

**Reply by HiWenHao** [reply]
新年快乐，感谢~♥

**Comment by qinsong729** [comment]
thank you

**Comment by Gracieful** [comment]
Is a new license available?

**Reply by endink** [reply]
20240226 is an updated license

**Comment by Amjid1234** [comment]


**Reply by Amjid1234** [reply]
any new license is there?

**Comment by Amjid1234** [comment]
hi @endink  we would like to purchase this plugin. how can we get it.
we already sent email to you. kindly give response.

**Reply by endink** [reply]
sorry, commerce version stop provide, but free version is available, download the license file in this topic

**Reply by endink** [reply]
new lincese file has been uploaded

**Reply by Amjid1234** [reply]
thank you so much :)

**Comment by lixch111111** [comment]
我下载最新的授权文件测试，不是到2024-06-26吗，为什么在ue5.1中我看授权过期时间2024-04-26？请问怎么回事？谢谢

**Reply by bilibilisizhuang** [reply]
我也是，即使是最新下载的仍然显示4.26过期，求助

**Reply by endink** [reply]
> 我也是，即使是最新下载的仍然显示4.26过期，求助

抱歉，文件有误，已经更新，下个版本计划免费，不再提供付费版本

**Reply by lixch111111** [reply]
非常感谢！！！

**Reply by bilibilisizhuang** [reply]
感谢感谢

**Comment by gigiwotou** [comment]
thanks man !!

**Comment by hammadfozi** [comment]
Hi, I have the old version for Unreal 5.1

My mediapipe4u plugin version is 2022.0.1

My license file is expired. Can you please send me another license file.

**Reply by endink** [reply]
Sorry, there is currently no support for UE versions earlier than `5.3`,  Support may be provided in the future, sorry again.

**Comment by yangyaqi-2025** [comment]
求求新的许可证感谢

**Reply by endink** [reply]
https://github.com/endink/Mediapipe4u-plugin/releases

每月这自动更新

**Comment by manoopazurestudies-design** [comment]
New license please? Please tell which time zone 8 o clock

**Reply by endink** [reply]
https://github.com/endink/Mediapipe4u-plugin/releases/tag/license-auto-release
